src/common/KernelBase.cpp

```diff
@@ -77,6 +77,13 @@ KernelBase::KernelBase(KernelID kid, const RunParams& params)
                                            CALI_ATTR_ASVALUE |
                                            CALI_ATTR_AGGREGATABLE |
                                            CALI_ATTR_SKIP_EVENTS);
+  for (unsigned i = 0; i < FeatureID::NumFeatures; ++i) {
+    std::string feature = getFeatureName(static_cast<FeatureID>(i));
```

Comment:
Can you pull out fid into a var to be consistent with the coding below?

Reply:
Done. [cabcbb3](https://github.com/LLNL/RAJAPerf/pull/456/commits/cabcbb3afedad19725e63014b4ed4b8be686f521)